nat-lab/tests/utils/bindings.py

```diff
@@ -0,0 +1,49 @@
+from typing import Callable, Optional
```

Comment:
It's going to have utilities for more of the bindings than just features, so I wouldn't make the name more specific, and since it's already in the `utils` folder, I think adding `helpers` to the name is a bit redundant